ls_0/index.html

```diff
@@ -0,0 +1,380 @@
+                            <span>59</span>
+                            <span>Min</span>
+                        </div>
+                    </div>
+                    <img src="/ls_0/src/images/why-us-photo.svg" alt="photo">
+                </div>
+                <a href="#" class="title-btn">Become a Sponsore</a>
+            </section>
+
+            <section>
+                <div class="block-line">
+                    <h2>Which one do you prefere ?</h2>
+                    <div class="line"></div>
+                </div>
+                <p>Laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi </p>
+                <p> beatae vitae dicta sunt explicabo. </p>
+            </section>
+
+            <div class="square-line">
+                <img src="/ls_0/src/images/Slide-left.svg" alt="arrow">
+                <img src="/ls_0/src/images/square-graphic.svg" alt="graphic">
+                <img src="/ls_0/src/images/square-design.svg" alt="graphic">
+                <img src="/ls_0/src/images/square-marketing.svg" alt="graphic">
+                <img src="/ls_0/src/images/square-education.svg" alt="graphic">
+                <img src="/ls_0/src/images/Slid-right.svg" alt="arrow">
+            </div>
+
+            <section class="customers">
+                <div class="customers-left">
+                    <img src="/ls_0/src/images/our-customers-blue.svg" alt="blue">
+                    <img src="/ls_0/src/images/our-customer-photo.svg" alt="photo">
+                </div>
+                <div class="customers-right">
+                    <div class="block-line">
+                        <h2>Which one do you prefere ?</h2>
+                        <div class="line"></div>
+                        <p>Sed ut perspiciatis unde omnis iste natus error sit </p>
+                        <p> voluptatem accusantium doloremque</p>
+
+                        <p> Laudantium, totam rem aperiam, eaque ipsa quae ab illoi </p>
+                        <p> nventore veritatis et quasi architecto beatae vitae dicta</p>
+                        <p> sunt explicabo. </p>
+                        <h4>Stephanie Magion</h4>
+                    </div>
+                    <img src="/ls_0/src/images/dots.svg" alt="dots">
+                </div>
+            </section>
+
+            <section class="block-started">
+                <div class="block-started-left">
+                    <h6>IT’S FAST, FREE AND VERY EASY!</h6>
+                    <h3>Ready to get started?</h3>
+                    <p>Sed ut perspiciatis unde omnis iste natus error sit </p>
+                    <p> voluptatem accusantium doloremque</p>
+
+                    <p> Laudantium, totam rem aperiam, eaque ipsa quae ab illoi </p>
+                    <p> nventore veritatis et quasi architecto beatae vitae dicta</p>
+                    <p> sunt explicabo. </p>
+                </div>
+                <a href="#" class="title-btn">Apply Now</a>
```

Comment:
"Get started" and "Apply now" - its an anchor link to the form (name, phone, email)